### PR TITLE
Adding several items to the cookie block list. Fixes several issues

### PR DESCRIPTION
**Pull request by cooperq**
CC @Hainish @ghostwords

### CODE REVIEW

File: doc/sample_cookieblocklist.txt [hunk 1]
```diff
@@ -537,9 +540,11 @@ wistia.net
 wistia.com
 sslstatic.wix.com
 gatherer.wizards.com
+s.w.org
```
Comment (ghostwords) [inline doc/sample_cookieblocklist.txt]: This one is missing from sample_cookieblocklist_legacy.txt

**Comment by ghostwords** [comment]
As far as technical issues go, there are a few differences between the regular and the "legacy" lists. One of these is new (s.w.org), the rest have been there before. It also looks like there is a typo in one of the suffixes in the legacy list. After normalizing and diffing:
```diff
84d83
< gr-assets.com
90a90
> gr-assets.com
377c377
< opf.ooyala.com^third-party
---
> opf.ooyala.com
414a415,417
> api.razorpay.com
> checkout.razorpay.com
> razorpay.com
540c543
< w.org
---
> s.w.org
```
What are these lists used for? Why does the legacy list exist? It seems like the legacy list is (or should be) just the regular list with some extra markup.

File: doc/sample_cookieblocklist_legacy.txt [hunk 2]
```diff
@@ -534,9 +537,11 @@
 @@||wistia.com^$third-party
 @@||sslstatic.wix.com^$third-party
 @@||gatherer.wizards.com^$third-party
+@@||w.org^$third-party
```
Comment (ghostwords) [inline doc/sample_cookieblocklist_legacy.txt]: Ah, should this be `s.w.org`?

**Comment by ghostwords** [comment]
Ideally we'd remove the duplication and update all consumers of this list to figure it out automatically. Failing that, we should add automatic linting to catch typos in the future.

**Comment by ghostwords** [comment]
Also, how come `w.org` is already in https://www.eff.org/files/cookieblocklist.txt when it's being added in this pull request?

**Comment by cooperq** [comment]
@ghostwords this should be pretty set now. Lets look at it tomorrow

**Pull request merged**